src/components/DataArchives/DataArchives.js

```diff
@@ -155,14 +157,9 @@ function DataArchives(props) {
           className={classes.dlFirst}
         >
           <Grid item className={classes.dt}>
-            <a
-              className={classes.link}
-              href="http://api.sensors.africa/v1/sensor/%7BsensorID%7D/"
-              target="_blank"
-              rel="noopener noreferrer"
-            >
+            <Typography variant="body2" component="p">
```

Comment:
Is the plan to remove the links or to change the URL?

Reply:
The component basically documents what each of the API endpoints achieves, and with the changes that make all the endpoints needing `Authorization`, keeping these links mean,
1. Having an API endpoint directly in html means exposing the `api Token` which as highlighted above, we're moving away from
2. Since there's no auth implementation currently, the links redirect to `DRF` unauthorised  error page. I don't think this is the experience we'd want.

Open to suggestions though!

Reply:
Links shouldn't point to APIs but to docs e.g. <https://api.sensors.africa/docs/#v2-sensors-list>